dev/swift/protoc/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM swift:5.7
+
+COPY protoc.sh /opt/protoc
+
+RUN apt-get -q update && \
+    apt-get -y install protobuf-compiler unzip curl \
+    && rm -r /var/lib/apt/lists/*
```

Comment:
Might want to align the && to either be on the beginning or ends of the lines for readability. Otherwise LGTM